software/control/gui_hcs.py

```diff
@@ -528,6 +532,14 @@ def setupLayout(self):
         layout.addWidget(self.sampleSettingsWidget)
         layout.addWidget(self.navigationViewer)
 
+        # Add performance mode toggle button
```

Comment:
the code sorta tells us this already.  For comments, a good rule is that you should use them to explain non-obvious things and/or provide critical context.

For instance, here you could have:
```
# live_only_mode is a config mode that sets up the gui to only show the live view.  In that case, the performance
# mode toggle doesn't do anything since its purpose is to toggle on/off the non-live views.  So we hide the
# performance mode toggle when the system boots in live_only_mode
```